back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -202,7 +202,10 @@ def selected_locale(omniauth_params)
 
     locales = AppConfiguration.instance.settings.dig('core', 'locales')
     selected_locale = omniauth_params['sso_pathname'].split('/', 2)[1].split('/')[0]
+
     return selected_locale if selected_locale != locales.first && locales.include?(selected_locale)
+
+    return locales.first
```

Comment:
Just wondering why you added this? As looking at the comments for the method, it seems to be specific on returning nil, though not sure why

Reply:
Wondering if this is causing the failing test

Reply:
Basically, I was encountering a bug where if you where starting the flow from `localhost:3000/en`, it would work, but if you started from `localhost:3000` (not sure why the locale even disappeared) everything would crash. Seemed a bit excessive to make everything crash just because the flow was started from a URL without a locale. Couldn't really imagine why it would need to work like that

Reply:
But happy to remove it again if it fixes the test